src/highrl/envs/robot_env.py

```diff
@@ -168,6 +171,16 @@ def step(self, action: List) -> Tuple:
         )
 
         self.episode_reward += self.reward
+        self.robot_reward_writer.add_scalar(
```

Comment:
Same here with the hardcoding.

Reply:
I do not understand which part exactly is in need of change?

Reply:
Done